stdlib/public/RemoteInspection/TypeLowering.cpp

```diff
@@ -1563,6 +1555,18 @@ const ReferenceTypeInfo *TypeConverter::getReferenceTypeInfo(
   return TI;
 }
 
+std::string TypeConverter::getLastError() {
+  if (!LastError.first)
+    return {};
+  std::stringstream s;
+  s << LastError.first;
+  if (LastError.second)
```

Comment:
This will sometimes print with a space (`markInvalid("No field descriptor: ", T);`), and sometimes without a space (`markInvalid("No TypeInfo for superclass", Superclass);`).  I think this function should control the formatting, and callers just pass an error message + the object, without worrying about how they will be concatenated.